examples/reference-entity-model/input/jsonCustomersWithRelatedEntities/customer1WithRelatedEntities.json

```diff
@@ -0,0 +1,88 @@
+{
+  "CustomerID": 301,
+  "Name": {
+    "FirstName": "Cynthia",
+    "LastName": "Waters"
+  },
+  "nicknames": [
+    "Cindy"
+  ],
+  "Email": "Cynthia.Waters@gmail.com",
+  "Address": [
+    {
+      "Shipping": {
+        "Street": "Abbey Court",
+        "Street2": "More info",
+        "City": "Columbus",
+        "State": "Ohio",
+        "Postal": "43210-0241"
+      }
+    },
+    {
+      "Billing": {
+        "Street": "Keys Crescent Ln",
+        "City": "Cincinnati",
+        "State": "Ohio",
+        "Postal": "45206-6783"
+      }
+    }
+  ],
+  "Phone": "(740) 407-2081",
+  "Created": "04/03/2017",
+  "Status": "Old Customer",
+  "SSN": "333-33-3333",
+  "Orders": [
+    {
+      "OrderId": "2002",
+      "DateAndTime":"2020/08/14-07:45:22",
+      "FirstName": "Audrey",
```

Comment:
Let's remove data that isn't going to be mapped here; it adds complexity without any value. I think it's good to remove some of the customer properties as well that we don't care about here - e.g. phone / created / status / ssn / email. Do keep nicknames though, as that will be interesting to include in a SQL query.